tick/inference/hawkes_conditional_law.py

```diff
@@ -430,7 +445,7 @@ def _lexical_to_index(self, i, j, l):
         """
         return self._ijl2index[i][j][l]
 
-    def incremental_fit(self, realization, compute=True):
+    def incremental_fit(self, realization, T=None, compute=False):
```

Comment:
I understand the need to be consistent across the library, however the whole point of `incremental_fit` with wiener-hopf is to average the conditional laws and then to compute a final estimate. With moderately high dimension the `compute` is quite slow so doing it at every update is very costly. So if `compute=True` is needed for consistency, at least we should warn the user I think...